browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -0,0 +1,63 @@
+<link rel="import" href="chrome://resources/html/polymer.html">
+
+<link rel="import" href="chrome://resources/html/i18n_behavior.html">
+<link rel="import" href="chrome://resources/html/md_select_css.html">
+<link rel="import" href="default_brave_shields_browser_proxy.html">
+<link rel="import" href="../settings_shared_css.html">
+<link rel="import" href="../settings_vars_css.html">
+
+<dom-module id="settings-default-brave-shields-page">
+  <template>
+    <style include="settings-shared md-select iron-flex">
+    </style>
+    <div class="settings-box">
+      <div class="start">$i18n{adControlLabel}</div>
+      <select id="adControlType" class="md-select"
+          on-change="onAdControlChange_">
+        <template is="dom-repeat" items="[[adControlTypes_]]">
+          <option value="[[item.value]]"
+              selected="[[controlEqual(item.value, adControlType_)]]">
+            [[item.name]]
+          </option>
+        </template>
+      </select>
+    </div>
+    <div class="settings-box">
+      <div class="start">$i18n{cookieControlLabel}</div>
+      <select id="cookieControlType" class="md-select"
+          on-change="onCookieControlChange_">
+        <template is="dom-repeat" items="[[cookieControlTypes_]]">
+          <option value="[[item.value]]"
+              selected="[[controlEqual(item.value, cookieControlType_)]]">
+            [[item.name]]
+          </option>
+        </template>
+      </select>
+    </div>
+    <div class="settings-box">
+      <div class="start">$i18n{fingerprintingControlLabel}</div>
+      <select id="fingerprintingControlType" class="md-select"
+          on-change="onFingerprintingControlChange_">
+        <template is="dom-repeat" items="[[fingerprintingControlTypes_]]">
+          <option value="[[item.value]]"
+              selected="[[controlEqual(item.value, fingerprintingControlType_)]]">
+            [[item.name]]
+          </option>
+        </template>
+      </select>
+    </div>
+    <settings-toggle-button id="httpsEverywhereControlType"
+        pref="{{prefs.brave.https_everywhere_default}}"
```

Comment:
Correct me if I'm wrong, but wouldn't this key need to be added to the whitelist of pref keys that appear in the `settingsPrivate` JS api? A Brave override for this doesn't yet appear to exist so I added one in #454 here: https://github.com/brave/brave-core/pull/454/files#diff-3971ac612305df099f1e78861ce8b884R16

Just wondering if there's something I'm missing and this works here without the whitelist addition @bbondy...

Reply:
Possibly, I did have a setting not found error but I haven't looked into why yet.

Reply:
I had the same error until I implemented the whitelist

Reply:
@petemill does that whitelist get propagated to `CreateIncognitoPrefService`? If so, those entries will be persisted to disk (using the `JsonPrefStore`)

Reply:
worked like a charm, thx @petemill